packages/actions-global/src/actions/handle-action-error.ts

```diff
@@ -1,16 +1,41 @@
+import { Dispatch } from 'redux';
 import {
   STATUS_CODES,
   LOGOUT_REASONS,
+  TStatusCode,
+  TAppNotificationApiError,
 } from '@commercetools-frontend/constants';
 import browserHistory from '@commercetools-frontend/browser-history';
 import showApiErrorNotification from './show-api-error-notification';
 import showUnexpectedErrorNotification from './show-unexpected-error-notification';
 
-export default function handleActionError(error, source) {
-  return dispatch => {
+type ActionError =
```

Comment:
This shows how "inconsistent" the error argument is. I hope I got the typing correct but we should really decide what type or error we want to pass around.